src/main/java/network/radicle/tools/github/utils/FileUtils.java

```diff
@@ -0,0 +1,79 @@
+package network.radicle.tools.github.utils;
+
+import org.apache.tika.Tika;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Base64;
+
+public class FileUtils {
```

Comment:
Ideally this should be refactored to a new service

Reply:
@JChrist Fixed. Could you review the changes please?